app/src/main/java/org/jellyfin/androidtv/ui/livetv/LiveTvGuideActivity.java

```diff
@@ -622,6 +623,8 @@ private LinearLayout getProgramRow(List<BaseItemDto> programs, String channelId)
             int slot = 0;
             do {
                 BaseItemDto empty = new BaseItemDto();
+                empty.setId(UUID.randomUUID().toString());
+                empty.setType("FOLDER");
```

Comment:
^ this might look stupid (because it does) but legacy code uses BaseItemDto's for messaging between activities for some stupid reason. SDK's BaseItemDto requires a valid UUID and type set so we need to add those here.